src/main/scala/nl.knaw.dans.easy.deposit/DataFiles.scala

```diff
@@ -107,11 +156,12 @@ case class DataFiles(bag: DansBag) extends DebugEnhancedLogging {
    * @return `true` if a new file was created, `false` if an existing file was overwritten
    */
   def write(is: InputStream, path: Path): Try[Boolean] = {
-    val fileExists = (bag.data / path.toString).exists
+    val fileExists = (uploadRoot / path.toString).exists
+    val pathInDataDir = bag.data.relativize(uploadRoot / path.toString)
     for {
-      _ <- if (fileExists) bag.removePayloadFile(path)
+      _ <- if (fileExists) cleanUp(pathInDataDir) // TODO see fetchFiles in StagedFilesTarget
```

Comment:
what does this TODO mean? I don't know what `StagedFilesTarget` is, neither can I find any other reference to it

Reply:
...somewhat later...
O wait, `StagedFilesTarget` is a file you deleted... Perhaps the TODO is no longer relevant?